xpk.py

```diff
@@ -1316,11 +1376,13 @@ def run_gke_node_pool_create_command(args, system) -> int:
         f' {capacity_args}'
         ' --scopes=storage-full,gke-default'
         ' --enable-gvnic --max-pods-per-node 15'
-        f' {args.custom_tpu_nodepool_arguments}'
     )
     if system.accelerator_type == AcceleratorType['TPU']:
+      command += (' --placement-type=COMPACT ')
       command += (f' --tpu-topology={system.topology}')
+      command += (f' {args.custom_tpu_nodepool_arguments}')
```

Comment:
Nice catch on `custom_tpu_nodepool_arguments`.

We will need a custom_device_nodepool_arguments to support the same feature across all device types. I will create a bug for this.

Reply:
Thank you!